raster-foundry/raster-foundry
# fixes contrast in color correction and filters, cleans up datepicker

## Overview
There were contrast issues on all three themes pertaining to the advanced color correction panel and scene filter panel. Beyond fixing these contrast issues, I spent some time to clean up other interactions.

- Cleaned up the UI for the date picker
- In advanced color correction, each label appearing next checkbox will now activate the checkbox when clicked.
- Each "select box" for the various dropdowns in the filters panel should look roughly the same now.
- The labels for each section in the filter panel should now be sitting above its respective filter (rather than to the left).

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~
~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
~- [ ] Any new SQL strings have tests~

### Demo

<img width="396" alt="screen shot 2018-11-05 at 11 22 40 am" src="https://user-images.githubusercontent.com/1928955/48012756-bd72f280-e0f0-11e8-8679-7dc3ba5b7428.png">
<img width="400" alt="screen shot 2018-11-05 at 11 23 16 am" src="https://user-images.githubusercontent.com/1928955/48012757-bd72f280-e0f0-11e8-8282-16704f81aa7f.png">
<img width="414" alt="screen shot 2018-11-05 at 11 52 32 am" src="https://user-images.githubusercontent.com/1928955/48013013-573a9f80-e0f1-11e8-922d-c034a999ca0a.png">
<img width="397" alt="screen shot 2018-11-05 at 11 25 44 am" src="https://user-images.githubusercontent.com/1928955/48012759-bd72f280-e0f0-11e8-9f25-640c22c8faef.png">
<img width="402" alt="screen shot 2018-11-05 at 11 27 40 am" src="https://user-images.githubusercontent.com/1928955/48012760-bd72f280-e0f0-11e8-8949-99180048dda8.png">
<img width="397" alt="screen shot 2018-11-05 at 11 35 20 am" src="https://user-images.githubusercontent.com/1928955/48012761-bd72f280-e0f0-11e8-89f2-d8011e97f53a.png">

## Testing Instructions

 * Loop through each theme to make sure the following lines up with the above screens:
   * Project browse filters panel
   * Project advanced color correction pane

Closes #4243
